### PR TITLE
Integrate aztec beta9

### PR DESCRIPTION
**Pull request by SergioEstevao**
Integrates Aztec pod beta 9 on the app.

To test:
 - Activate the new editor
 - Give it a spin to see if there is any major issues.
 - Test copy paste of block of text inside the editor, because this was one of the areas that was causing us trouble.

Needs review: @jleandroperez  and @astralbodies please give it spin around!

### CODE REVIEW

**Comment by SergioEstevao** [comment]
@astralbodies and @jleandroperez want to give it a last look before I release Aztec beta 9?

File: Podfile [hunk 1]
```diff
@@ -54,7 +54,7 @@ target 'WordPress' do
   pod 'NSURL+IDN', '0.3'
   pod 'WPMediaPicker', '0.18'
   pod 'WordPress-iOS-Editor', '1.9.3'
-  pod 'WordPress-Aztec-iOS', '1.0.0-beta.8.1'
+  pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :branch => 'release/1.0.0-beta.9'
```
Comment (SergioEstevao) [inline Podfile]: This is just for testing before I launch the official pod. If all is ok I release the pod and update this reference.

**Comment by astralbodies** [comment]
I'm not sure it's related to this PR but when the app is restored via state restoration the image I had pasted in previously doesn't show up:

**Comment by astralbodies** [comment]
The image attachment flashes between the placeholder and the image - I assume that's not expected?

https://www.screencast.com/t/YiYanUE7rj

**Comment by astralbodies** [comment]
The image tag is technically incorrect - it should be closed with `` I think?

```
<img src="https://ardwptest1.files.wordpress.com/2017/07/image-2.jpg" height="2002" class="wp-image-273" width="3000">
```

**Comment by astralbodies** [comment]
The image shows a different aspect ratio/zoom level when it's first placed, then after it flashes back to the placeholder, then flashes back to the final image in the editor. In the video you'll see space on the left and right of the image, then the placeholder, then the image shows full width.

https://www.screencast.com/t/TomXcuIw3Z2

**Comment by SergioEstevao** [comment]
@astralbodies all valid points and I think all of them where already present before. Will open some new tickets for it.

**Comment by SergioEstevao** [comment]
> The image attachment flashes between the placeholder and the image - I assume that's not expected?

This comes from us switching between the placeholder and the new remote image on the server. We will need to optimize the code to not go back to the default placeholder when we already have one value placeholder.

**Comment by SergioEstevao** [comment]
> The image tag is technically incorrect - it should be closed with  I think?

Libxml used to does this and it looks we are also doing it, @jleandroperez any reason for this?

**Comment by astralbodies** [comment]
@SergioEstevao Merge conflict 😄

**Comment by SergioEstevao** [comment]
@astralbodies fixed the conflict.

**Comment by astralbodies** [comment]
:shipit:

**Pull request merged**